course/src/main/java/in/testpress/course/fragments/ContentLoadingFragment.kt

```diff
@@ -151,7 +151,12 @@ class ContentFragmentFactory {
                 }
                 "Html" -> HtmlContentFragment()
                 "Notes" -> HtmlContentFragment()
-                "VideoConference" -> VideoConferenceFragment()
+                "VideoConference" -> {
+                    if (content.video != null && content.videoConference?.showRecordedVideo == true) {
```

Comment:
This can be extracted as a method in Domaincontent so that intent will be easier to understand